content/docs/user-guide/interactivity/user-interface/animating/tweener-system/tweener-lua-code.md

```diff
@@ -58,4 +59,4 @@ self.ScriptedEntityTweener:StartAnimation
 	}
 ```
 
-For the parameter descriptions, see [Tweener Supported Components](tweener-components/).
+For the parameter descriptions, see [Tweener Supported Components](./tweener-components).
```

Comment:
This is the format that we're using for links, with a forward slash (`/`) at the end and if within the same directory, no leading `./`. Can you revert that to the other links as well?
```suggestion
For the parameter descriptions, see [Tweener Supported Components](tweener-components/).
```

Reply:
Thank you for the suggestions, yes, I'll change all links.  If you want something like this unified site-wide at some point, it would be an easy matter for me to PR an automated edit.

Reply:
@FiniteStateGit I was informed that not having `./` isn't an official standard in O3DE docs. There's also an argument that eliminating `./` would indicate to eliminate `../`, which would enforce absolute links. For now, our docs may have `./` or not, and `../` or absolute links, until something has been decided by the SIG officially. You do not need to remove `./` from links, I apologize for the initial misdirection. If you already made that change, no need to change it. Thanks again!

Reply:
Another bit of info I learned and want to clarify, we are using `/` at the end of paths to **directories**, not files. Since tweener-components is a file, I believe you formatted it correctly originally :) 

You don't need to make the changes I originally suggested here, and I will continue reviewing your PR as it is.